CS/CASESTUDY1/VehicleDetailPrinter.java

```diff
@@ -0,0 +1,11 @@
+
+public class VehicleDetailPrinter {
+
+	public static String printVehicleSpecification(Vehicle vehicle)
+	{
+		return vehicle.toString();    // print details of vehicle
```

Comment:
ACTION: According to the requirement "This method should return the generic attributes of Vehicle class as string" but your method will return all the attributes based on the type of vehicle instance.For eg- If it is instance of bike class it will return all attributes of vehicle along with the bike attributes.